client/js/components/procedure/SegmentsList/SegmentsList.vue

```diff
@@ -8,11 +8,14 @@
 </license>
 
 <template>
-  <div>
+  <div
+    :class="{ 'top-0 left-0 w-full h-full fixed z-[2200] bg-white': isFullscreen }">
     <dp-sticky-element
       border
-      class="u-pv-0_5">
-      <div class="flex items-start space-inline-s">
+      class="py-4"
+      :class="{ 'fixed top-0 left-0 w-full h-[15%] px-4': isFullscreen }">
```

Comment:
I think its not a good Idea to work with percentages. suming the sticky element has have a height of 150px, we could calculate with css the datatableheight like `calc(100% - 150px)` (but I dont know how/if tailwind support such things)